streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/ClientTagAwareStandbyTaskAssignor.java

```diff
@@ -211,17 +194,32 @@ static void assignStandbyTasksToClientsWithDifferentTags(final ConstrainedPriori
                 break;
             }
 
-            clientStates.get(clientOnUnusedTagDimensions).assignStandby(activeTaskId);
-
+            final ClientState clientStateOnUsedTagDimensions = clientStates.get(clientOnUnusedTagDimensions);
             countOfUsedClients++;
             numRemainingStandbys--;
 
+            log.debug("Assigning {} out of {} standby tasks for an active task [{}] with client tags {}. " +
+                      "Standby task client tags are {}.",
+                      numberOfStandbyClients - numRemainingStandbys, numberOfStandbyClients, activeTaskId,
+                      clientStates.get(activeTaskClient).clientTags(), clientStateOnUsedTagDimensions.clientTags());
+
+            clientStateOnUsedTagDimensions.assignStandby(activeTaskId);
             lastUsedClient = clientOnUnusedTagDimensions;
         } while (numRemainingStandbys > 0);
 
         if (numRemainingStandbys > 0) {
             pendingStandbyTasksToClientId.put(activeTaskId, activeTaskClient);
             tasksToRemainingStandbys.put(activeTaskId, numRemainingStandbys);
+            log.warn("Rack aware standby task assignment was not able to assign {} of {} standby tasks for the " +
```

Comment:
Added logging based on the previous PR comment https://github.com/apache/kafka/pull/10851#discussion_r817629787